### DISCUSSION TITLE
Deployed website will not load.

### DISCUSSION THREAD

**Discussion by JHebenstreit48** [community/community#132856] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get my website I made for video game to load. I fixed all the issues with the CSS not showing up and certain pages not linking to others after using "Open in Default Browser" through VS Code. However, when I try to open the deployed page I see the following and not sure why the site will not load. 
![GitHub Pages issue](https://github.com/user-attachments/assets/d5d61f30-4b6f-46b2-93cd-72214efb4602)

I can post the repo link if needed.

**Comment by knarrff** [comment]
Hi @JHebenstreit48 

It is helpful if you could, next time, also give the URL of the repo so it easier for people to take a look. I assume for this case it  is https://github.com/JHebenstreit48/ashphalt-legends-unite.

When you look at the output from the build process (https://github.com/JHebenstreit48/ashphalt-legends-unite/actions/runs/10017100120/job/27690882143) and expand the portion about jekyll, you see that only README.md is rendered to html. That is the content you see on https://jhebenstreit48.github.io/ashphalt-legends-unite/. The files in the HTML subdirectory are already html files and thus are not touched by jekyll. They are still served though, here: https://jhebenstreit48.github.io/ashphalt-legends-unite/HTML/index.html . I suppose that is the content you are looking for?

If you do want to have them show up without the "HTML" in the URL I suppose you could move them one directory up, either in the repository itself or as part of building the pages.

**Comment by JHebenstreit48** [comment]
I ended up just changing the file structure back to hopefully fix it. I had moved index.html into a folder called HTML. It was not in the default file pile. I just did that will all of my main tabs.

**Reply by knarrff** [reply, marked as answer]
Hi @JHebenstreit48

I am glad moving the html files out of HTML/ helped, I can also now see the repo under the URL you wanted. If you can too, it may be time to mark the question as answered.

Good luck with your game!
